Apps/GatewayApi/test/unit/Services.Test/DependentServiceTests.cs

```diff
@@ -74,6 +76,26 @@ public void GetDependents()
             }
         }
 
+        /// <summary>
+        /// GetDependents by date - Happy Path.
+        /// </summary>
+        [Fact]
+        public void GetDependentsByDate()
```

Comment:
You'll get a few more lines of coverage if you include a test for the path that returns an error.